components/nimbus/src/tests/test_behavior.rs

```diff
@@ -1306,11 +1317,7 @@ mod event_store_tests {
             store.query(event_id, Interval::Weeks, 52, 0, EventQueryType::LastSeen)?
         );
         assert_eq!(
-            0.0,
-            store.query(event_id, Interval::Months, 12, 0, EventQueryType::LastSeen)?
-        );
```

Comment:
This is the test that was broken.

It was failing because [when the test was written](https://github.com/mozilla/application-services/pull/5431), 1 week ago and now were both in the same month.

But it's the 27th today and 1 week ago would be 20th of the same month. Is that right?

Yes, because the `event_store` considers 1 month == 28 days.

Yesterday, was the 84th day of the year, so from today, for a week, this test would fail.

Reply:
Is there a way we could stub out `Utc::now()`? We could set the test up with a static date if the rolling dates prove to be a problem